src/CreateReceipt.java

```diff
@@ -0,0 +1,45 @@
+import java.text.DecimalFormat;
```

Comment:
Look at proposed changes, tests would need to be adjusted to account for corrected totals